test/test.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Tests to verify stub behaviors
+ */
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  UDPDevice,
+  MockNetwork,
+  RemoteAddressInfo,
+} from '../platform/mock-radio';
+import {
+  MockLocalHomePlatform,
+  UDPScanConfig,
+  ScanState,
+} from '../platform/mock-local-home-platform';
+
+// Tests a UDP identify flow end-to-end
+test('udp-device-connects', (t) => {
+  // First, create a scan configuration
+  const scanConfig = new UDPScanConfig(
+    ScanState.Unprovisioned,
+    '255.255.255.255',
+    3311,
+    3312,
+    'A5A5A5A5'
+  );
+
+  const mockNetwork = new MockNetwork();
+
+  // Mock the Local Home Platform
+  const mockLocalHomePlatform = new MockLocalHomePlatform(mockNetwork, [
+    scanConfig,
+  ]);
+
+  // Mock a UDP Device
+  const mockDevice = new UDPDevice();
+
+  const deviceId = 'test-device-id';
+  // Device data that mock device sends back
+
+  const discoveryPort = 12345;
+  const discoveryData = JSON.stringify({
+    id: 'test-device-id',
+    model: '2',
+    hw_rev: '0.0.1',
+    fw_rev: '1.2.3',
+    channels: [discoveryPort],
+  });
+
+  // Sample device response
+  mockDevice.setUDPMessageAction((msg: Buffer, rinfo: RemoteAddressInfo) => {
```

Comment:
does this need to be that dynamic? Maybe we could simply have a triggertIdentify that accept the discoveryBuffer to the be send to the app?

Reply:
Opened issue #27